Makefile

```diff
@@ -317,7 +317,7 @@ generate-k8s-api:
 	$(call generate_k8s_api_deepcopy,github.com/cilium/cilium/pkg,"policy:api")
 	$(call generate_k8s_api_deepcopy,github.com/cilium/cilium,"pkg:loadbalancer")
 	$(call generate_k8s_api_deepcopy,github.com/cilium/cilium,"pkg:k8s")
-	$(call generate_k8s_api_deepcopy,github.com/cilium/cilium,"pkg:node")
+	$(call generate_k8s_api_deepcopy,github.com/cilium/cilium,"pkg:service")
```

Comment:
Why this addition? I don't think it's need it. :thinking:

Reply:
rebase error :(